omnigibson/object_states/robot_related_states.py

```diff
@@ -50,13 +51,14 @@ def _get_value(self, obj):
 #         return not body_ids.isdisjoint(robot.states[ObjectsInFOVOfRobot].get_value())
 
 
-# class ObjectsInFOVOfRobot(AbsoluteObjectState):
-#     def _get_value(self):
-#         # Pass the FOV through the instance-to-body ID mapping.
-#         seg = self.simulator.renderer.render_single_robot_camera(self.obj, modes="ins_seg")[0][:, :, 0]
-#         seg = np.round(seg * MAX_INSTANCE_COUNT).astype(int)
-#         body_ids = self.simulator.renderer.get_pb_ids_for_instance_ids(seg)
-
-#         # Pixels that don't contain an object are marked -1 but we don't want to include that
-#         # as a body ID.
-#         return set(np.unique(body_ids)) - {-1}
\ No newline at end of file
+class ObjectsInFOVOfRobot(AbsoluteObjectState, RobotStateMixin):
+    def _get_value(self):
+        if not any(isinstance(sensor, VisionSensor) for sensor in self.robot.sensors.values()):
+            raise ValueError("No vision sensors found on robot.")
+        prim_paths = []
+        paths_to_exclude = ['BACKGROUND', 'UNLABELLED']
```

Comment:
make set instead of list (usually better performance, tho for a 2-length list it's probably trivial)